main/src/cgeo/geocaching/maps/MapSettingsUtils.java

```diff
@@ -158,6 +159,50 @@ public void setValue() {
         }
     }
 
+    private static class ToggleButtonWrapper<T> {
```

Comment:
@fm-sys in my understanding this would replace the ToggleButtonGroup, right? Can you externalize it so it can be used in other places as well (e.g. the filter activity...)? Or should I do that later myself?

Reply:
First of all, it's a replacement for `ButtonController` (an internal class of `MapSettingsUtils`). But if it looks suitable for you, I can externalize it as well...

Reply:
I see now that it is not a view itself.